tests/functional/tester/cluster_run.go

```diff
@@ -212,7 +182,7 @@ func (clus *Cluster) doTestCase(t *testing.T, fa Case) {
 		zap.String("desc", fa.Desc()),
 	)
 	if err := fa.Inject(clus); err != nil {
-		t.Fatalf("injection error: %v", err)
+		clus.failed(t, fmt.Errorf("injection error, %w", err))
```

Comment:
My two cents: I think `:` is better than `,`, since it is error wrapper. 
In most of go program, it is common to use `xxxx: %w`.

REF: https://go.dev/blog/go1.13-errors

Reply:
thx & updated.